addons/mrp/models/mrp_workorder.py

```diff
@@ -203,6 +203,12 @@ def _compute_allowed_lots_domain(self):
     def name_get(self):
         return [(wo.id, "%s - %s - %s" % (wo.production_id.name, wo.product_id.name, wo.name)) for wo in self]
 
+    @api.multi
+    def unlink(self):
+        # Removes references to workorder to avoid Validation Error
+        self.workorder_id = False
```

Comment:
self could be multi, not sure you can assign like that on multiple records. To check